Library/Koala/Koala.swift

```diff
@@ -515,7 +515,7 @@ public final class Koala {
     self.track(event: "Changed Pledge Amount", properties: props)
   }
 
-  public func trackSelectedShippingDestination(project: Project,
+  public func trackSelectedShippingDestination(_ project: Project,
```

Comment:
yeah def, but wanna do that pass in a separate pr to minimize the diff of the compatibility stuff